lib/EarlyFunctionAnalysis/EarlyFunctionAnalysis.cpp

```diff
@@ -1788,9 +1884,15 @@ bool EarlyFunctionAnalysis::runOnModule(Module &M) {
   for (const auto &[_, Node] : BasicBlockNodeMap)
     RootNode->addSuccessor(Node);
 
+  // The intraprocedural analysis will be scheduled only for those functions
+  // which have `Invalid` as type.
   UniquedQueue<BasicBlockNode *> CFEPQueue;
   for (auto *Node : llvm::post_order(&CG)) {
-    if (Node != RootNode)
+    if (Node == RootNode)
+      continue;
+
+    auto &Function = Binary.Functions[getBasicBlockPC(Node->BB)];
```

Comment:
`operator[]` will create the entry if absent, which is not what we want. Switch to `at()`.

Reply:
Correct, forgot this.